tests/parm/field_table/field_table_hafs_thompson

```diff
@@ -0,0 +1,65 @@
+# added by FRE: sphum must be present in atmos
```

Comment:
What is the difference between this file and `https://github.com/ufs-community/ufs-weather-model/blob/develop/tests/parm/field_table/field_table_thompson_aero_tke`?

Reply:
field_table_hafs_thompson is the same as field_table_thompson_noaero_tke. I just want to hold a spot for future changes of field_table_hafs_thompson. Note that field_table_hafs is the same as field_table_gfsv16.